Aztec/Classes/TextKit/TextView.swift

```diff
@@ -301,6 +301,33 @@ open class TextView: UITextView {
     }
 
 
+    /// Get a list of format identifiers of the Typing Attributes.
+    ///
+    /// - Returns: A list of identifiers.
+    ///
+    open func formatIdentifiersForTypingAttributes() -> [String] {
```

Comment:
I just realized we're using `[String]` instead of `[FormattingIdentifier]` throughout our code.  Sounds like something we should change (unless there's a reason not to, let me know).